src/assets/_project/_blocks/layout/footer/footer-feedback.html

```diff
@@ -8,7 +8,7 @@ <h3>Website feedback</h3>
             </a>
             <div id="qg-footer-feedback" class="qg-footer-feedback collapse">
                 <div class="qg-footer-feedback-content">
-                    <form id="qg-page-feedback-form" method="post" action="https://www.qld.gov.au/contact-us/thank-you?SQ_ASSET_CONTENTS_RAW" class="form" data-recaptcha="true" novalidate>
+                    <form id="qg-page-feedback-form" method="post" action="https://smartservice.qld.gov.au/services/submissions/email/feedback/feedback-v2" class="form" data-recaptcha="true" data-action="feedback" novalidate>
```

Comment:
www.smartserviece.qld.gov.au. without the www it will fail.

Reply:
in the recent commit, I have made this change